extensions/wikia/TemplateClassification/TemplateClassificationHooks.class.php

```diff
@@ -0,0 +1,23 @@
+<?php
+class TemplateClassificationHooks {
+
+	/**
+	 * Adds assets for AuthPages on each Oasis pageview
+	 *
+	 * @param {String} $skin
```

Comment:
Type of a var in `@param` tag should follow the PHPDoc style: http://www.phpdoc.org/docs/latest/references/phpdoc/tags/param.html